src/types/webhook.ts

```diff
@@ -48,8 +48,10 @@ export type UpdateWebhook = Partial<CreateWebhook> & {
 }
 
 export type WebhookEventPayload = {
-  /** Primary identifier of the resource that was notified */
+  /** @deprecated id of the entity related to the event */
```

Comment:
We could improve the `@deprecated` message by adding what field you should use. ie.
```
/**
 * id of the entity related to the event
 * @deprecated This field is deprecated. Use the `eventId` parameter to identify the resource instead.
 */
```